README.md

```diff
@@ -263,6 +263,28 @@ API clients (Web UIs, Mobile, etc.)
 * https://github.com/openMF has more related proejcts
 
 
+Generate Apache Fineract API Client
```

Comment:
@vorburger Actually, I already added the description in the README.md. 
But I can definitely create a new file, as you suggested, under `docs/developers/swagger/client.md` and link it with the main README.md